freqtrade/templates/strategy_analysis_example.ipynb

```diff
@@ -215,13 +215,18 @@
     "stats = load_backtest_stats(backtest_dir)\n",
     "strategy_stats = stats['strategy'][strategy]\n",
     "\n",
+    "dates = []\n",
+    "profits = []\n",
+    "for date_profit in strategy_stats['daily_profit']:\n",
```

Comment:
When doing updates to the notebook, please always run the 2 jupyter commands afterwards (as documented [here](https://www.freqtrade.io/en/stable/developer/#updating-example-notebooks)).

that'll keep the notebook "clean" (it's clean now too - but this makes sure of it) - and also makes it available as documentation should someone want to copy it into an existing notebook.

Reply:
I ran the commands but I'm not seeing any changes in any files, am I doing something wrong?

Reply:
Should I do anything else for the PR?

Reply:
Oh I see you ran the commands to get the docs updated, sorry I couldn't look at this earlier.